build.gradle

```diff
@@ -118,6 +121,7 @@ dependencies {
 
     // ZipArchive dependencies used for integration tests
     zipArchive group: 'org.opensearch.plugin', name:'opensearch-ml-plugin', version: "${opensearch_build}"
+    zipArchive group: 'org.opensearch.plugin', name:'opensearch-sql-plugin', version: "${version}"
```

Comment:
Shouldn't all the plugins have the same version with `opensearch_build`?

Reply:
makes sense